src/features/pages/options/accumulator/accumulators_faq.module.scss

```diff
@@ -0,0 +1,3 @@
+.accordion_border{
+    border-bottom: 2px solid #F6F7F8 ;
+}
```

Comment:
Can't we use tailwind for it? and why are we not using token for color name?

Reply:
Will use token for color name but for the tailwind I tried putting it in the component from quill many times and it's not working. After a conversation with Meenu I will put it as scss for now until we fix it